masterlist.yaml

```
@@ -22793,7 +22793,7 @@ plugins:
         subs:
           - 'CC Tundra Homestead'
           - '[Whiterun Exteriors Patch Collection](https://www.nexusmods.com/skyrimspecialedition/mods/78920/)'
-        condition: 'active("cceejsse001-hstead.esm") and not active("Ryns Whiterun City Limits - (Iggath )?Tundra Homestead Patch\.esp")'
+        condition: 'active("cceejsse001-hstead.esm") and (not active("Ryns Whiterun City Limits - (Iggath )?Tundra Homestead Patch\.esp") or not active("JKs Ryns Whiterun Exterior - CC Tundra Homestead patch.esp"))'
```

@loicreynier, this condition would result in the message only being hidden if one of the patches in the regex ***&*** the JK combo patch is active (or if the CC mod isn't active of course), which presumably isn't what you intended. This condition would hide it if at least one of any of the three patches is active:
```suggestion
        condition: 'active("cceejsse001-hstead.esm") and not active("Ryns Whiterun City Limits - (Iggath )?Tundra Homestead Patch\.esp") and not active("JKs Ryns Whiterun Exterior - CC Tundra Homestead patch.esp")'
```

---

Oh yes I forgot about the regex, thanks.